services/user/Registry/README.md

```diff
@@ -0,0 +1,18 @@
+# Psibase Registry
+
+Steps to run the app in dev mode:
+
+```sh
+# after building psibase at the root of the repo
+cd build
+./psinode --http-timeout 10 db
```

Comment:
Is the value of 4 in the default config template too short? If we need to clutter docs like this without explanation, it would be better just to increase the default.

Reply:
not sure, I cant test anymore because we removed the UI... 
I pretty much just copied and pasted how I run my cmd, let me remove this arg.

Reply:
Not sure why we raced to delete the UI, it wouldn't have been terrible to have two UIs for a while, especially while Registry was still in development